src/main/java/by/exadel/internship/entity/Form.java

```diff
@@ -45,11 +47,13 @@
     @Column(name = "fm_skype")
     private String skype;
 
-    @Column(name = "fm_country")
-    private String country;
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
I would not set CascadeType.ALL because if we will delete form we will delete country

Reply:
done